src/theme/Navbar/Logo/styles.module.css

```diff
@@ -0,0 +1,3 @@
+.navbar__logo img{
+  width: 21px !important;
```

Comment:
is `!important` necessary here? We try to avoid those whenever possible.

Reply:
I have removed `!important`. 👍🏼